config/initializers/feature_toggle.rb

```diff
@@ -20,4 +20,10 @@ def self.location_toggle_enabled?
 
     false
   end
+
+  def self.new_locations_table_layout_enabled?
+    return true unless Rails.env.production?
+
+    false
+  end
```

Comment:
I know this is just duplicating previous methods (I did the same for a scheme_toggle one) but these could probably be 
```suggestion
def self.location_toggle_enabled?
    !Rails.env.production?
  end
```
etc.